sympy/solvers/solveset.py

```diff
@@ -327,7 +327,12 @@ def _invert_complex(f, g_ys, symbol):
                                imageset(Lambda(n, f.inverse()(n)), g_ys), symbol)
 
     if isinstance(f, exp):
-        if isinstance(g_ys, FiniteSet):
+        x = Symbol('x')
+        if (len((f.args)) < 2) and (f.args == (exp(x),)):
+            print(len((f.args)))
```

Comment:
you will want to remove the `print` before committing

Reply:
yeah ,
my apologies